
Epoch 1/30


277/277 [==============================] - ETA: 0s - loss: 0.6355 - mean_absolute_error: 0.0511
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
277/277 [==============================] - 8s 22ms/step - loss: 0.6355 - mean_absolute_error: 0.0511 - val_loss: 0.6315 - val_mean_absolute_error: 0.0305
Epoch 2/30

273/277 [============================>.] - ETA: 0s - loss: 0.6242 - mean_absolute_error: 0.0296
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
277/277 [==============================] - 5s 17ms/step - loss: 0.6240 - mean_absolute_error: 0.0296 - val_loss: 0.6300 - val_mean_absolute_error: 0.0249
Epoch 3/30
 29/277 [==>...........................] - ETA: 2s - loss: 0.6217 - mean_absolute_error: 0.0271


274/277 [============================>.] - ETA: 0s - loss: 0.6226 - mean_absolute_error: 0.0236
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
277/277 [==============================] - 5s 19ms/step - loss: 0.6227 - mean_absolute_error: 0.0236 - val_loss: 0.6291 - val_mean_absolute_error: 0.0204
Epoch 4/30

277/277 [==============================] - 3s 12ms/step - loss: 0.6221 - mean_absolute_error: 0.0207 - val_loss: 0.6292 - val_mean_absolute_error: 0.0229
Epoch 5/30

273/277 [============================>.] - ETA: 0s - loss: 0.6219 - mean_absolute_error: 0.0193
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
277/277 [==============================] - 5s 19ms/step - loss: 0.6218 - mean_absolute_error: 0.0193 - val_loss: 0.6284 - val_mean_absolute_error: 0.0169
Epoch 6/30
 55/277 [====>.........................] - ETA: 2s - loss: 0.6179 - mean_absolute_error: 0.0170

217/277 [======================>.......] - ETA: 0s - loss: 0.6215 - mean_absolute_error: 0.0169
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

275/277 [============================>.] - ETA: 0s - loss: 0.6215 - mean_absolute_error: 0.0169
277/277 [==============================] - 5s 18ms/step - loss: 0.6214 - mean_absolute_error: 0.0169 - val_loss: 0.6283 - val_mean_absolute_error: 0.0165
Epoch 7/30

276/277 [============================>.] - ETA: 0s - loss: 0.6214 - mean_absolute_error: 0.0170
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
277/277 [==============================] - 9s 34ms/step - loss: 0.6213 - mean_absolute_error: 0.0170 - val_loss: 0.6282 - val_mean_absolute_error: 0.0159
Epoch 8/30

274/277 [============================>.] - ETA: 0s - loss: 0.6214 - mean_absolute_error: 0.0163
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
277/277 [==============================] - 5s 18ms/step - loss: 0.6212 - mean_absolute_error: 0.0163 - val_loss: 0.6281 - val_mean_absolute_error: 0.0150
Epoch 9/30

277/277 [==============================] - 4s 13ms/step - loss: 0.6212 - mean_absolute_error: 0.0160 - val_loss: 0.6287 - val_mean_absolute_error: 0.0212
Epoch 10/30

274/277 [============================>.] - ETA: 0s - loss: 0.6211 - mean_absolute_error: 0.0161
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
277/277 [==============================] - 5s 18ms/step - loss: 0.6211 - mean_absolute_error: 0.0161 - val_loss: 0.6280 - val_mean_absolute_error: 0.0148
Epoch 11/30
 72/277 [======>.......................] - ETA: 2s - loss: 0.6225 - mean_absolute_error: 0.0151

238/277 [========================>.....] - ETA: 0s - loss: 0.6211 - mean_absolute_error: 0.0150
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

276/277 [============================>.] - ETA: 0s - loss: 0.6209 - mean_absolute_error: 0.0149
277/277 [==============================] - 5s 18ms/step - loss: 0.6210 - mean_absolute_error: 0.0149 - val_loss: 0.6279 - val_mean_absolute_error: 0.0140
Epoch 12/30
277/277 [==============================] - 4s 13ms/step - loss: 0.6211 - mean_absolute_error: 0.0162 - val_loss: 0.6280 - val_mean_absolute_error: 0.0150
Epoch 13/30

277/277 [==============================] - 4s 13ms/step - loss: 0.6211 - mean_absolute_error: 0.0161 - val_loss: 0.6280 - val_mean_absolute_error: 0.0152
Epoch 14/30


274/277 [============================>.] - ETA: 0s - loss: 0.6209 - mean_absolute_error: 0.0165
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
277/277 [==============================] - 5s 19ms/step - loss: 0.6212 - mean_absolute_error: 0.0165 - val_loss: 0.6278 - val_mean_absolute_error: 0.0136
Epoch 15/30
277/277 [==============================] - 4s 13ms/step - loss: 0.6209 - mean_absolute_error: 0.0141 - val_loss: 0.6279 - val_mean_absolute_error: 0.0151
Epoch 16/30


274/277 [============================>.] - ETA: 0s - loss: 0.6212 - mean_absolute_error: 0.0153
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
277/277 [==============================] - 5s 19ms/step - loss: 0.6210 - mean_absolute_error: 0.0153 - val_loss: 0.6278 - val_mean_absolute_error: 0.0136
Epoch 17/30
 74/277 [=======>......................] - ETA: 2s - loss: 0.6192 - mean_absolute_error: 0.0140


273/277 [============================>.] - ETA: 0s - loss: 0.6207 - mean_absolute_error: 0.0137
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
277/277 [==============================] - 5s 18ms/step - loss: 0.6208 - mean_absolute_error: 0.0137 - val_loss: 0.6277 - val_mean_absolute_error: 0.0131
Epoch 18/30
277/277 [==============================] - 4s 13ms/step - loss: 0.6208 - mean_absolute_error: 0.0138 - val_loss: 0.6287 - val_mean_absolute_error: 0.0234
Epoch 19/30

277/277 [==============================] - 4s 13ms/step - loss: 0.6209 - mean_absolute_error: 0.0144 - val_loss: 0.6279 - val_mean_absolute_error: 0.0147
Epoch 20/30

277/277 [==============================] - 4s 13ms/step - loss: 0.6208 - mean_absolute_error: 0.0135 - val_loss: 0.6279 - val_mean_absolute_error: 0.0158
Epoch 21/30

277/277 [==============================] - 4s 14ms/step - loss: 0.6208 - mean_absolute_error: 0.0139 - val_loss: 0.6283 - val_mean_absolute_error: 0.0203
Epoch 22/30

277/277 [==============================] - 3s 12ms/step - loss: 0.6208 - mean_absolute_error: 0.0141 - val_loss: 0.6278 - val_mean_absolute_error: 0.0137
Epoch 23/30

270/277 [============================>.] - ETA: 0s - loss: 0.6208 - mean_absolute_error: 0.0139
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
277/277 [==============================] - 5s 18ms/step - loss: 0.6208 - mean_absolute_error: 0.0139 - val_loss: 0.6277 - val_mean_absolute_error: 0.0133
Epoch 24/30
 25/277 [=>............................] - ETA: 3s - loss: 0.6314 - mean_absolute_error: 0.0125


274/277 [============================>.] - ETA: 0s - loss: 0.6207 - mean_absolute_error: 0.0129
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
277/277 [==============================] - 5s 18ms/step - loss: 0.6207 - mean_absolute_error: 0.0129 - val_loss: 0.6276 - val_mean_absolute_error: 0.0125
Epoch 25/30

277/277 [==============================] - 4s 13ms/step - loss: 0.6207 - mean_absolute_error: 0.0136 - val_loss: 0.6276 - val_mean_absolute_error: 0.0124
Epoch 26/30

274/277 [============================>.] - ETA: 0s - loss: 0.6212 - mean_absolute_error: 0.0129
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
277/277 [==============================] - 5s 17ms/step - loss: 0.6207 - mean_absolute_error: 0.0129 - val_loss: 0.6276 - val_mean_absolute_error: 0.0122
Epoch 27/30

277/277 [==============================] - 4s 13ms/step - loss: 0.6208 - mean_absolute_error: 0.0140 - val_loss: 0.6289 - val_mean_absolute_error: 0.0223
Epoch 28/30
277/277 [==============================] - 4s 13ms/step - loss: 0.6208 - mean_absolute_error: 0.0137 - val_loss: 0.6277 - val_mean_absolute_error: 0.0133
Epoch 29/30


272/277 [============================>.] - ETA: 0s - loss: 0.6203 - mean_absolute_error: 0.0128
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
277/277 [==============================] - 5s 19ms/step - loss: 0.6206 - mean_absolute_error: 0.0128 - val_loss: 0.6275 - val_mean_absolute_error: 0.0118
Epoch 30/30
 67/277 [======>.......................] - ETA: 2s - loss: 0.6244 - mean_absolute_error: 0.0123


277/277 [==============================] - 3s 12ms/step - loss: 0.6206 - mean_absolute_error: 0.0125 - val_loss: 0.6276 - val_mean_absolute_error: 0.0125